config/routes.rb

```diff
@@ -1,2 +1,4 @@
 Rails.application.routes.draw do
+  get 'products/index'
```

Comment:
redundant line of code, please remove it

Reply:
done